### PR TITLE
Side bar fix issue #1

### LINKED ISSUE #1: Inconsistent Sidebar Styles
- All titles in the sidebar should have `monospace` style font
- All bullets should be unfilled circles.
### Screenshot:

sidebar styles
#### Steps to reproduce:
1. as admin login and create at least one article
2. proceed to localhost:3000 and view the article 
3. on the right side observe the font for Archives differs from the other headers
4. on the right side observe that the bullet points under archives are filled circles
#### Expected behavior:
1. Archives font should be monotype
2. Archives bullets should use unfilled-circle

### PR DESCRIPTION
**Pull request by ilterates**
Fixed  #1 

[Resolves #1]

### CODE REVIEW

**Comment by tgaff** [comment]
@ilterates Could you reference the issue number that this fixes?

**Comment by ilterates** [comment]
Added issue number.

File: app/assets/stylesheets/user-styles.css [hunk 1]
```diff
@@ -28,10 +28,10 @@ img.right {
   margin: -8px; padding: 8px;
 }
 
-#sidebar .sidebar-title {
+#sidebar .sidebar_title {
```
Comment (tgaff) [inline app/assets/stylesheets/user-styles.css]: @ilterates We're using `.sidebar-title` in a lot of places in this app.  We can't remove it!

File: app/assets/stylesheets/user-styles.css [hunk 2]
```diff
@@ -28,10 +28,10 @@ img.right {
...
   font-family: monospace;
 }
 
-#sidebar .sidebar-body li {
+#sidebar .sidebar_body li {
```
Comment (tgaff) [inline app/assets/stylesheets/user-styles.css]: ditto above

**Comment by tgaff** [comment]
`git grep sidebar-title`

File: lib/popular_sidebar/app/views/popular_sidebar/_content.html.erb [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <h3 class="sidebar-title"><%= sidebar.title %></h3>
 <div class="sidemenu-content">
-  <ul class="popular_sidebar">
+  <ul class="popular-sidebar">
```
Comment (tgaff) [inline lib/popular_sidebar/app/views/popular_sidebar/_content.html.erb]: This class doesn't exist in our css.  Is it related to what you're fixing?

**Comment by ilterates** [comment]
Problem was the class name for archieves was spelled with and "_" rather than "-"

**Comment by tgaff** [comment]
Nice!

**Pull request closed without merging**